db/tests/patient_details_migration.migrationTest.js

```diff
@@ -0,0 +1,431 @@
+import { execSync } from "child_process";
+import AppContainer from "../../src/containers/AppContainer";
+import { setupWardWithinHospitalAndTrust } from "../../src/testUtils/factories";
+import { COMPLETE, SCHEDULED } from "../../src/helpers/visitStatus";
+
+describe("patient details migration", () => {
+  const container = AppContainer.getInstance();
+  let date;
+
+  beforeEach(() => {
+    date = new Date();
+    date.setDate(date.getDate() + 1);
+  });
+
+  it("migrates the patient details data from the visits table to the patient_details table", async () => {
+    const downAllMigrations = execSync("npm run dbmigratetest reset");
+    console.log(downAllMigrations.toString());
+
+    // run all migrations up until 24/07/2020 (which is just before the patient details migration)
+    const runMigrationsToBeforePatientDetailsMigration = execSync(
+      "npm run dbmigratetest up 20200724000000"
+    );
```

Comment:
Feels like there might be a clearer way to express this, is it not possible to reference the previous migration by name?

Reply:
I think specifying the migration by name just run that specific migration, rather than running all the migrations up to that point. It's possible to specify the number of migrations to run, but not sure that makes it any clearer.